src/meta/src/controller/catalog.rs

```diff
@@ -2395,6 +2396,35 @@ impl CatalogController {
         Ok(version)
     }
 
+    pub async fn list_stream_job_desc_for_telemetry(
+        &self,
+    ) -> MetaResult<Vec<MetaTelemetryJobDesc>> {
+        let inner = self.inner.read().await;
+        let info: Vec<(TableId, Property)> = Table::find()
```

Comment:
This part seems to have been forgotten to be changed? And in catalog manager only table and mview have been selected, we should maintain logical consistency.